### DISCUSSION TITLE
Meeting to discuss about the TriliumNext organization

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Trilium#6066] [General]
Hi,

I would like to set up an audio/video meeting so that we can discuss the first steps that we can take to start working on the Trilium Next project, but also to discuss together some big ideas that you might have.

Feel free to open as many discussion topics as you want even before this meeting. I'm particularily interested in those ideas that would have significant impact on the future of the project, such as maybe altering the tech stack (or refactoring) or adding a nice new feature (e.g. be more Notion-like). I believe now is a good time to discuss it since there is currently no pressure on us delivering anything for now and we don't have yet zadam's experience on the project, we might as well make the changes now.

As a first step, feel free to leave a comment here indicating that you'd like to participate in this meeting and a possible time slot availability on your side (including the time zone).

**Comment by zerebos** [comment]
I think a live meeting like this would be great for an initial brainstorming session and for getting on the same page. I think it's important for everyone that wants to be involved to at least have somewhat similar visions of how this fork should continue or else we'll be pulling the app in several different directions disjointedly. I also agree that any of the big ideas like refactoring the tech stack or massive features should be brought up and discussed as early as possible.

Of course I am very interested in a meeting like this. I can be available any week day (Mon-Fri) from 5pm - 2am EST, or pretty much anytime most weekends. It may be difficult to coordinate this yourself, Elian, so you could also consider a tool like https://www.when2meet.com/.

**Reply by eliandoran** [reply]
@rauenzi , sounds great.

Let's see as a first step who is actually interested in joining this meeting.

**Reply by AkechiShiro** [reply]
Interested in discussing more about : https://github.com/zadam/trilium/issues/4646

Available to speak in English CEST 8 P.M. - 12 P.M.

Especially the modular part (backend vs frontend), however I'm not familiar at all with the current codebase nor Trilium usage, just yet, but I'm planning on extensively using it.

**Comment by SiriusXT** [comment]
I'm interested in discussing Trilium Next.  But I can not speak English.  If there is a text discussion, I can participate.  Available time period: AOE time, 14:00-04:00.

**Reply by eliandoran** [reply]
@SiriusXT , thanks for bringing this up. For the very first meeting, I'd love it to be an audio meeting.
I will be sure to take meeting of minutes and share them with you afterwards.
Also I'd encourage you to join @rauenzi 's Discord https://discord.gg/eTaTXUgcBr and we can chat there.

**Reply by meichthys** [reply]
On a similar note, I can listen in, but I would not speak.

**Comment by rom1dep** [comment]
Count me in as a curious listener :) I'll join if I can make it

**Comment by rodrigoalexandrecosta** [comment]
I also would like to join the conversation. I'm available after 5 pm BRT (which right now is 3 pm EST, 9 pm CEST and 8 pm GMT), but I also can make something like 5 or 6 am BRT until around 9 am BRT. I'm also basically free in the weekends and would only need some previous "let me know" so I can move things around. Will join the Discord soon.

**Reply by root-hal9000** [reply]
Opa, I have a compatriot here it seems. Valeu Rodrigo!

**Comment by meichthys** [comment]
I would listen but not speak. I would be available any weeknight after 8:30pm eat

Dont schedule around me tho, I'd be glad with a transcript or summary of talking points.

**Comment by j-tirelli** [comment]
I'm generally available from 5pm-12am EST.

**Comment by meichthys** [comment]
@eliandoran We should make this happen sooner rather than later. We have lots of interest in helping out but no direction in where we want to go first. We need goals and tasks that can be delegated to all these new "contributors". We should have both software development tasks as well as documentation, issue triaging, and community outreach tasks.

**Reply by eliandoran** [reply]
@meichthys , you're absolutely right. Let's follow @maphew 's suggestion and let's get started. :)

**Comment by Micosilent** [comment]
Count me in for the meeting any week day from 18:00 to 21:00 CET

**Comment by root-hal9000** [comment]
I am late to the party, but would be happy to join. @eliandoran shall we just poll it and make a time? Which also brings me a random question: anyone have a FOSS alternative to Doodle Polls suggestion I suppose?

EDIT: Just saw the discord link, joined there as well

**Reply by meichthys** [reply]
There is a built-in poll feature in github which isn't Foss, but is easy. 
Maybe [Cal.com](https://cal.com/) is what you're looking for in FOSS?

**Reply by root-hal9000** [reply]
Ah thanks - yeah, I pretty much want to self host or use a decent an alternative to doodle polls for coordinating meeting time slots - personally (or for us here if that works too)

**Reply by meichthys** [reply]
Definitely Cal.com for scheduling. It's pretty slick.

**Reply by root-hal9000** [reply]
thanks bud!

**Reply by XaverStiensmeier** [reply]
Just to throw in a second option:
https://nuudel.digitalcourage.de/
> What is special about this [Framadate]((https://framagit.org/framasoft/framadate/framadate)) instance?

> This server runs on our own hardware and does not do any user tracking. We deactivated access logging both in the [Framadate](https://framagit.org/framasoft/framadate/framadate) software and in the webserver. No IP addresses are stored, only the content you enter into the forms. You can also decide not to specify an email address; in that case please make sure to enter anything ending in “@invalid” in the email field. If you decide to do this, you should create a bookmark for the admin pages of your polls.

digitalcourage is a well known german privacy and digital rights organisation - I love them but I am not associated.

**Comment by rmkni** [comment]
TLDR; instead of sharing your availability, suggest a meeting time that works for you and others can join to discuss the threads on github. ;)
---
Here is a suggested structure for organizing discussions:
- Utilize the github discussions panel as a list of topics to discuss.
- If a topic for discussion is not listed, create a new one.
- During the discussion, take notes and put them in the thread for everyone to see key takeaways.

Afterwards, one takes an initiative to schedule a meeting session (e.g. "Hi everyone, I will be hosting a meeting on Tuesday at 6pm USD+7") and encourage participation through visible announcements (such as discord pings, mailing list updates, or new discussion topics on github).

Even if only a few people attend a session, they can still engage in productive conversations, share ideas, and subsequently regroup with different individuals for further discussions in a more organic manner.

**Comment by maphew** [comment]
I created a poll to collect time zones stats - https://github.com/orgs/TriliumNext/discussions/58, thinking that from that we might be able to pick a recurring day and time. It might be easier to coalesce on _"first friday of every month at XX o'clock"_ than periodic one-offs. _Scheduling is hard ;-)_

**Comment by root-hal9000** [comment]
@TriliumNext/everyone
Hey everyone, we're trying to narrow down a time when most people are available over the next few weeks. This is not a poll for regular availability for recurring meetings! For now, this is for a one time meeting! We can decide later if we want recurring ones, or maybe individual teams might have recurring ones, etc. So take that into account since I assume that opens more potential times than if people were selecting for recurring availability

Please select all the times you're available to meet in this scheduling poll:
https://www.when2meet.com/?25414487-FaSGo
you can enter a username/name to record your preferences, and the password is optional for the sake of being able to come back and edit, so probably a good idea to save one if you're OK with that (the password is just for the poll, not a registration for the website).

**Reply by eliandoran** [reply]
@root-hal9000 , nice. Thank you for taking the initiative.
Seems it's actually going to be harder to have everyone at the same time due to the very different timezone.

**Reply by root-hal9000** [reply]
np. Yeah, it's not looking great so far, but let's see. If it turns out that i have to just one time take a meeting in the middle of the night, so be it haha

**Reply by maphew** [reply]
I filled in my easiest-to-make-it-hours. If it happens outside those selections I still might be able to.

**Reply by CobriMediaJulien** [reply]
Do we have a date yet?

**Reply by root-hal9000** [reply]
> Do we have a date yet?

yes, I just added as another comment:

Sunday, July 7th, at 5:00 PM UTC

You can use this link to join it with Jitsi:
https://meet.jit.si/moderated/260e53a5d58765ec8ba11c60fc02566dc8b85b07a0f5f90c8f5c4fba2a1e2d22

**Comment by root-hal9000** [comment]
Hey everyone, we have a meeting time set:
Sunday, July 7th, at 5:00 PM UTC

You can use this link to join it with Jitsi:
https://meet.jit.si/moderated/260e53a5d58765ec8ba11c60fc02566dc8b85b07a0f5f90c8f5c4fba2a1e2d22

**Comment by root-hal9000** [comment]
Sorry if too much extra noise, but I made a new discussion thread TriliumNext/Notes#187  to ensure visibility for others who were not following this here
